var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -227,6 +230,7 @@ def edit(self, spec, prefix):
             'nvhpc': ['-fast'],
             'cray': ['-O2'],
             'xl': ['-O3'],
+            'aocc': ['-O3', '-ffast-math', '-Hx,47,0x10000008', ],
```

Comment:
What is the:
```
-Hx,47,0x10000008
```
flag for?

Reply:
"-Hx,47,0x10000008" flag is fix for known issue (precision loss) with AOCC v2.2 and v2.3. it will be resolved in upcoming AOCC releases.

Reply:
@amd-toolchain-support Can you please add a comment saying that above the lines where the flag appears?

Reply:
@alalazo , added comments as required.